source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -62,6 +62,8 @@ class CAdvisorMetricsAPIClient
   @@telemetryCpuMetricTimeTracker = DateTime.now.to_time.to_i
   @@telemetryMemoryMetricTimeTracker = DateTime.now.to_time.to_i
   @@telemetryPVKubeSystemMetricsTimeTracker = DateTime.now.to_time.to_i
+  @@userAgent = "ama-logs/#{ENV['AGENT_VERSION'].nil? ? '0.0.0' : ENV['AGENT_VERSION']} (#{ENV['OS_TYPE'].nil? ? 'linux' : ENV['OS_TYPE']}; Ruby #{RUBY_PLATFORM})"
```

Comment:
wrap this code to function and use that function. This will be cleaner.

Reply:
i wrote this here to avoid multiple function calls